tasks/config/sample.ts

```diff
@@ -0,0 +1,36 @@
+import Config from '../../lib/Config';
+import { isPlainObject } from '../../lib/utils/utils';
+import fs from 'fs';
+import path from 'path';
+
+export default async () => {
+  let result = '';
+
+  const recursivelyConvertJsonToToml = (json: any, prefix: string) => {
+    const nestedPairs: any = [];
+    const simplePairs: any = [];
+
+    if (!json) return;
+
+    Object.keys(json).sort().forEach((key: any) => {
+      if (Object.prototype.toString.call(json[key]) !== '[object Function]') {
+        (isPlainObject(json[key]) ? nestedPairs : simplePairs).push([key, json[key]]);
+      }
+    });
+
+    if (!(prefix === '' || simplePairs.length === 0)) {
+      result += '\n[' + prefix + ']\n';
+    }
+
+    simplePairs.forEach((value: any) => {
+      result += value[0] + ' = ' + value[1] + '\n';
```

Comment:
I think we need to handle two special cases here. The simpler one is if `typeof value[1] === 'string'` then we should surround `value[1]` with `""` in the result, because TOML expects strings to be in quotation marks.

Then we also need to check if the value is an array, in which case we need to use `[]`. We only have one array config property, `p2p.addresses` and it defaults to an empty array. I don't think we'll ever have default values in an array, so if you just want to detect if a value is an array using `isArray()` or something and then output `[]` that should be fine.

Reply:
done